packages/openstreetmap/test/pattern.test.mjs

```diff
@@ -7,7 +7,8 @@ const prefixes = ['', '//', 'www.', '//www.', 'https://', 'http://', 'https://ww
 const suffixes = [
   '/#map=8/46.195/-81.362',
   '/way/1147323572#map=8/46.195/-81.362',
-  '/search?whereami=1&query=52.147%2C104.106#map=8/46.195/-81.362'
+  '/search?whereami=1&query=52.147%2C104.106#map=8/46.195/-81.362',
+  '/?mlat=46.195&mlon=-81.362#map=8/46.195/-81.362'
```

Comment:
This was a new-to-me URL structure that came up in the original issue, so I'm adding it to the list.